source/exe/main_common.h

```diff
@@ -121,7 +121,7 @@ class MainCommon {
   PlatformImpl platform_impl_;
   Envoy::OptionsImpl options_;
   Event::RealTimeSystem real_time_system_;
-  DefaultTestHooks default_test_hooks_;
+  DefaultListenerHooks default_test_hooks_;
```

Comment:
rename this variable?

Reply:
Done (grepped for and renamed all other test_hooks variables as well).